groovy/method-handle.md

```diff
@@ -0,0 +1,16 @@
+# `.&` Method handle
+
+Instead of using anonymous classes or closures, you may use method handle by prefixing method with `&`.
+
+```groovy
+void onCreate(Bundle savedInstanceState) {
+    // ...
+    loginButton.onClickListener = this.&onLoginButtonClick
+}
+
+private void onLoginButtonClick(View view) {
+    // ...
+}
+```
+
+where `onClickListener` expects [`View.OnClickListener`](http://developer.android.com/reference/android/view/View.OnClickListener.html) interface.
```

Comment:
Nice one :)

You can use footnotes for links to external resources in Markdown:

```markdown
where `onClickListener` expects [`View.OnClickListener`] interface.

[`View.OnClickListener`]: http://developer.android.com/reference/android/view/View.OnClickListener.html
```

It helps keep the text more maintainable, as the content is separated from the links.